templates/vue/src/components/Lightbox/media/ActivityMedia/UserMultipleChoiceForm.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <b-form class="container" @submit="handleMultipleChoiceSubmit">
+    <b-form-group v-if="question.answerTypes.multipleChoice.hasMultipleAnswers">
+      <b-form-checkbox-group v-model="userSelectedCheckbox">
+        <user-choice-row
+          v-for="userChoiceRow in question.answerTypes.multipleChoice.checkboxArray"
+          :key="userChoiceRow.id"
+          :item="userChoiceRow"
+          :isCheckBox="question.answerTypes.multipleChoice.hasMultipleAnswers"
+          :hasImage="question.answerTypes.multipleChoice.useImages"
+          :data-qa="`user-choicerow-checkbox-${userChoiceRow.id}`"
+        ></user-choice-row>
+      </b-form-checkbox-group>
+      <!-- <p> You Selected: {{userSelectedCheckbox}}</p> -->
+    </b-form-group>
+
+    <b-form-group v-else>
+      <b-form-radio-group v-model="userSelectedRadio">
+        <user-choice-row
+          v-for="userChoiceRow in question.answerTypes.multipleChoice.radioArray"
+          :key="userChoiceRow.id"
+          :item="userChoiceRow"
+          :isCheckBox="question.answerTypes.multipleChoice.hasMultipleAnswers"
+          :hasImage="question.answerTypes.multipleChoice.useImages"
+          :data-qa="`user-choicerow-radio-${userChoiceRow.id}`"
+        ></user-choice-row>
+      </b-form-radio-group>
+      <!-- <p> You Selected: {{userSelectedRadio}}</p> -->
+    </b-form-group>
+    <b-form-invalid-feedback
+      :state="isAnswerValid"
+      style="clear:both"
+      data-qa="invalid-feedback"
+    >
+      Please Select a choice.
+    </b-form-invalid-feedback>
+    <b-button class="submit-btn mt-3" variant="primary" type="submit">
+      Submit
+    </b-button>
+  </b-form>
+</template>
+
+<script>
+import UserChoiceRow from "./UserChoiceRow"
+export default {
+  name: "user-multiple-choice-form",
+  components: {
+    UserChoiceRow,
+  },
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+    question: {
+      type: Object,
+      required: true,
+    },
+    answer: {
+      type: [String, Array],
+      required: true,
+    },
+  },
+  data() {
+    return {
+      isAnswerValid: true,
+      userSelectedCheckbox: [],
+      userSelectedRadio: null,
+      multipleChoiceAnswer: "",
+    }
+  },
+  computed: {
+    radioValidAnswerState() {
+      return Boolean(this.userSelectedRadio)
+    },
+    multipleChoiceId() {
+      return this.question.answers.multipleChoiceId
+    },
+  },
```

Comment:
Perhaps a computed property like
`multipleChoice() { return question.answerTypes.multipleChoice}` will make the template cleaner.

Reply:
This is not used anymore, so I will just delete it